tests/core/snapshots/test-scalars.md_10.snap

```diff
@@ -0,0 +1,27 @@
+---
+source: tests/core/spec.rs
+expression: response
+---
+{
+  "status": 200,
+  "headers": {
+    "content-type": "application/json"
+  },
+  "body": {
+    "data": null,
+    "errors": [
+      {
+        "message": "internal: invalid value for scalar \"PhoneNumber\", expected \"FieldValue::Value\"",
```

Comment:
This isn't a very meaningful error for the user.

Reply:
this is the error from async_graphql we don't specify it. All of the scalar errors looks like this rn